web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -440,7 +628,7 @@ Finally, since Wasp no longer requires you to separate your client source files
 as long as you keep all the source files in the `src/` directory.
 
 This section is optional, but if you didn't like the server/client
-separation, now's the perfect time to change it!
+separation, now's the perfect time to change it.
```

Comment:
I think we shouldn't use that much exclamation marks all the time. We also have a point about that in our writing guide: https://github.com/wasp-lang/wasp/blob/main/web/docs/writingguide.md#grammar

Reply:
Again, those are good rules, but let's not be overly strict with them, it kills the vibe. They are guidelines, not the law. Sentence here is quite excited: "Now's the perfect time" -> it is almost weird to not finish this with exclamation mark. And it is not like we have a ton of exclamation marks around so that we are using them all the time.

Reply:
Now that I'm rereading it, I would go even more neutral and say it's a good time to change it instead of perfect 😂

We decided to base our Writing guide on Vue's since we recognized them as an experienced bunch that had a lot of community input about their docs and they employ doc writers which know what they are doing.

I like the cheerful attitude in your written communication generally, but here I would go for more neutral tone 🤷🏻‍♂️